packages/syntax-core/src/SelectList/SelectList.tsx

```diff
@@ -98,7 +95,7 @@ const SelectList = ({
           onChange={onChange}
         >
           {placeholderText && (
-            <option disabled value="">
+            <option disabled value="" selected>
```

Comment:
@nathan-ong If we have a `selectedValue` and a placeholder, we want the `selectedValue` to be selected, not the `option`.

So I would change it to something like this:
```tsx
<select
  //
  value={
    placeholderText && !selectedValue ? placeholderText : selectedValue
  }
>
  {placeholderText && (
    <option disabled value={placeholderText}>
      {placeholderText}
    </option>
  )}
```

That way:
* If we have `placeholderText` but no `selectedValue`, the placeholder still gets selected
* If we have `placeholderText` and `selectedValue`, the correct option gets selected